pkgs/development/python-modules/django-simple-captcha/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, buildPythonPackage, mock, 
+django, django-ranged-response, pillow }:
+
+buildPythonPackage rec {
+  pname = "django-simple-captcha";
+  version = "0.5.6";
```

Comment:
There is a `0.5.11` version available, why not upgrade to it now (https://pypi.org/project/django-simple-captcha/)?

Reply:
Sure, I'll change it. The latest version doesn't work with some of the older django versions though. What is your policy on packaging multiple different versions (like in [django](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/python-packages.nix#L2536)'s case)? Can I package both `0.5.6` and `0.5.11`, making `0.5.11` the default one?

Reply:
Yes, please see ``pg8000`` and ``pg8000_1_12`` as an example.